apps/misc/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
 # Create your models here.
+
+class Initiative(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.TextField(max_length=5000)
+    thumbnail = models.ImageField(upload_to='../media/images/') 
```

Comment:
`upload_to` field already starts from the `media` folder and we need to provide a path here relative to `media` folder. So, `misc/initiatives` would work best here.